python/wdl/engine.py

```diff
@@ -1,12 +1,11 @@
 from wdl.binding import *
 from wdl.util import *
-import wdl.parser
 import re
 import subprocess
 import tempfile
 import uuid
-import json
-import sys
+import xml.etree.ElementTree as ETree
```

Comment:
ElementTree is part of the stdlib in python3

https://docs.python.org/3/library/xml.etree.elementtree.html#module-xml.etree.ElementTree